app/components/views/Offset/index.tsx

```diff
@@ -374,36 +374,13 @@ export const Offset = (props: Props) => {
             </Trans>
           </Text>
         </div>
+
         <div className={styles.offsetCard_ui}>
-          {/* Input Token */}
-          <DropdownWithModal
-            label="Pay with"
-            modalTitle="Select Token"
-            currentItem={selectedInputToken}
-            items={inputTokenItems}
-            isModalOpen={isInputTokenModalOpen}
-            onToggleModal={() => {
-              setInputTokenModalOpen((s) => !s);
-            }}
-            onItemSelect={handleSelectInputToken}
-          />
-          {/* Retire Token  */}
-          <DropdownWithModal
-            label="Select carbon offset token to retire"
-            modalTitle="Select Carbon Type"
-            currentItem={selectedRetirementToken}
-            items={retirementTokenItems}
-            isModalOpen={isRetireTokenModalOpen}
-            onToggleModal={() => {
-              setRetireTokenModalOpen((s) => !s);
-            }}
-            onItemSelect={handleSelectRetirementToken}
-          />
           <div className={styles.input}>
             <label>
-              <Text t="caption" color="lightest">
+              <Text t="caption" color="lighter">
                 <Trans id="offset.amount_in_tonnes">
-                  AMOUNT IN CARBON TONNES
+                  How many carbon tonnes would you like to retire?
```

Comment:
```suggestion
                  How many tonnes of carbon would you like to offset?
```

Wording tweak to illustrate the relationship between offset -> retirement better
1. enter quantity to "offset"
2. select which token to "retire"

Reply:
Done! :)